src/legacy/core_plugins/data/public/index.ts

```diff
@@ -25,25 +34,34 @@ class DataPlugin {
   private readonly indexPatterns: IndexPatternsService;
   private readonly search: SearchService;
   private readonly query: QueryService;
+  private readonly expressionExecutor: ExpressionExecutorService;
 
   constructor() {
     this.indexPatterns = new IndexPatternsService();
     this.query = new QueryService();
     this.search = new SearchService();
+    this.expressionExecutor = new ExpressionExecutorService();
   }
 
   public setup() {
     return {
       indexPatterns: this.indexPatterns.setup(),
       search: this.search.setup(),
       query: this.query.setup(),
+      expressionExecutor: this.expressionExecutor.setup(null, {
+        interpreter: {
+          getInterpreter,
```

Comment:
i think its ok to import these inside the interpreter service directly @lukeelmers  what do you think ?
as interpreter plugin will move in here as well (so this is pretty much the doing re-exporting we are doing in other data services)

Reply:
Talked with @ppisljar about this, currently the interpreter imports fail when not done on the top level. I will leave it like this for now and it can be removed when moving interpreter over